examples/color/named_colors.py

```diff
@@ -73,11 +81,27 @@ def plot_colortable(colors, title, sort_colors=True, emptycols=0):
 
     return fig
 
+#############################################################################
+# -----------
+# Base colors
+# -----------
+
 plot_colortable(mcolors.BASE_COLORS, "Base Colors",
                 sort_colors=False, emptycols=1)
+
+#############################################################################
+# ---------------
+# Tableau Palette
+# ---------------
+
 plot_colortable(mcolors.TABLEAU_COLORS, "Tableau Palette",
                 sort_colors=False, emptycols=2)
 
+#############################################################################
+# ----------
+# CSS Colors
+# ----------
+
 # sphinx_gallery_thumbnail_number = 3
 plot_colortable(mcolors.CSS4_COLORS, "CSS Colors")
 
```

Comment:
It's not here but in the rendered docs can the xkcd colors + commented out code be moved into its own section w/ a note about commented for brevity w/o attached image? 
 ![Screenshot_20220426-191925.jpg](https://user-images.githubusercontent.com/1300499/165408294-125eddf4-9b84-44a1-b35d-e248664bb9e8.jpg)

That way the note at the top can be removed but I think the xkcd support would be more obvious.

Reply:
Better?

https://output.circle-artifacts.com/output/job/35889340-cdb8-4400-8ca7-e70049220578/artifacts/0/doc/build/html/gallery/color/named_colors.html

I also removed the `.. contents::` because I noticed that the theme already puts the contents in the right sidebar.